pkgs/top-level/aliases.nix

```diff
@@ -1419,7 +1419,7 @@ mapAliases ({
   pybitmessage = throw "pybitmessage was removed from nixpkgs as it is stuck on python2"; # Added 2022-01-01
   pyext = throw "pyext was removed because it does not support python 3.11, is upstream unmaintained and was unused"; # Added 2022-11-21
   pygmentex = texlive.bin.pygmentex; # Added 2019-12-15
-  pyload = throw "pyload has been removed from nixpkgs, as it was unmaintained"; # Added 2021-03-21
+  pyload = throw "pyload has been removed from nixpkgs. Use pyload-ng instead."; # Added 2021-03-21; Edited 2023-07-09
```

Comment:
```suggestion
  pyload = throw "pyload has been removed from nixpkgs. Use pyload-ng instead."; # Added 2021-03-21
```

Reply:
:confounded: https://github.com/NixOS/nixpkgs/pull/242260#discussion_r1257383718

@AndersonTorres

Reply:
@SuperSandro2000 I believe this should be done because the crude "Added 2021-03-21" is false and/or at least misleading: the expression was originally added 2021-03-21, but it was modified 2023-07-09.

If a misleading information can be kept as comment, then the comment is utterly useless.

Reply:
This will probably break our automation script and the main purpose of the comment is to know when we can remove the entry again.